common/build-style/gem.sh

```diff
@@ -0,0 +1,23 @@
+#
+# This helper is for templates using gem files from RubyGems.
+#
+do_install() {
+	: ${gem_cmd:=gem}
+
+	local _GEMDIR
+
+	_GEMDIR="$($gem_cmd env gemdir)"
+
+	$gem_cmd install \
+		--local \
+		--install-dir ${DESTDIR}/${_GEMDIR} \
+		--bindir ${DESTDIR}/usr/bin \
+		--ignore-dependencies \
+		--verbose \
```

Comment:
Although I'm all for being verbose I think void usually doesn't use the verbose mode.

Reply:
```
$ gem install --install-dir /tmp/asciidoctor asciidoctor-1.5.4.gem
Successfully installed asciidoctor-1.5.4
1 gem installed
```

```

Reply:
Uh?

Reply:
Yeah sorry, the verbose output shows all the files to be installed